src/Scope.ts

```diff
@@ -264,10 +264,13 @@ export class Scope {
                 links.push({ item: nsContainersWithStatement?.namespaceStatements?.[0], file: nsContainersWithStatement?.file as BrsFile });
             }
         }
-
+        const fullNameLower = (containingNamespace ? `${containingNamespace}.${name}` : name).toLowerCase();
         const callable = this.getCallableByName(name);
```

Comment:
Get callable by name will find callables regardless of namespaces... we need to check the namespace after the fact